dbms/src/DataStreams/SharedQueryBlockInputStream.h

```diff
@@ -183,6 +183,16 @@ class SharedQueryBlockInputStream : public IProfilingBlockInputStream
             throw Exception(exception_msg);
     }
 
+    uint64_t collectCPUTimeNsImpl(bool /*is_root*/) override
+    {
+        uint64_t cpu_time_ns = 0;
```

Comment:
`is_root` is ignored because `SharedQueryBlockInputStream` should never be a root?

Reply:
comments added
```
        // SharedQueryBlockInputStream isn't a thread-runner, so is_thread_runner is ignored here.
        // But each of SharedQueryBlockInputStream's children is a thread-runner.
```